piicatcher/scanner.py

```diff
@@ -66,27 +66,27 @@ def scan(self, text):
 
 class ColumnNameScanner(Scanner):
     regex = {
-        PiiTypes.PERSON: re.compile(
+        ('customer_data', CustomerDataTypes.PERSON): re.compile(
```

Comment:
Categories can be added by updating this dictionary. Categories supported are:

- **customer_data**
- **customer_usage_data**
- **authentication_data**